.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "python.testing.unittestArgs": [
+        "-v",
+        "-s",
+        "./tests",
+        "-p",
+        "*_test.py"
+    ],
+    "python.testing.pytestEnabled": false,
+    "python.testing.unittestEnabled": true
```

Comment:
>python [](http://example.com/codeflow?start=5&length=6)

Seems this file is for code editor not for build process. Will it impact others if they also has similar settings file?

Reply:
This file is not required. Removed